pkgs/development/python-modules/pyicu/default.nix

```diff
@@ -7,8 +7,9 @@
 }:
 
 buildPythonPackage rec {
-  pname = "PyICU";
+  pname = "pyicu";
```

Comment:
It looks like this line needs to stay `PyICU` or the `fetchPypi` below should be changed to hardcode `PyICU`, since that is the casing on PyPI and the fetcher currently fails with the all-lowercase name

Reply:
Oh yeah, didn't notice due to caching. Will push a fix in a minute.